index.html

```diff
@@ -17,6 +17,7 @@
     <link rel="stylesheet" href="node_modules/animate.css/animate.min.css">
     <link href='https://fonts.googleapis.com/css?family=PT+Sans|Fredoka+One|Source+Code+Pro' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/style.css">
+    <script src="https://cdn.firebase.com/js/client/2.4.1/firebase.js"></script>
```

Comment:
What was the reason for putting this in `head` instead of with the other scripts at line 96?